### PR TITLE
self chat with browser

### PR DESCRIPTION
**Pull request by LaurinUB**
- Adding connection and disconnection for websocket
- Send and show messages on site and console

### CODE REVIEW

File: backend/package.json
Comment (philippschwarzmueller) [inline backend/package.json:1]: do we really need the nestjs/patform-socket.io, nestjs/websockets and socket.io?
Look to me as if the first one would be enough

File: frontend/src/components/chatwindow/Chatwindow.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from "react";
+import Input from "../input/Input";
+import Button from "../button/Button";
+import { io } from "socket.io-client";
+
+interface IChatwindow {}
+
+
+const Chatwindow: React.FC<IChatwindow> = (props: IChatwindow) => {
```
Comment (philippschwarzmueller) [inline frontend/src/components/chatwindow/Chatwindow.tsx:9]: the props are never used nor is the interface being used, i'd remove it for now

File: frontend/src/components/chatwindow/properties.ts
Comment (philippschwarzmueller) [inline frontend/src/components/chatwindow/properties.ts:1]: this is from the gamewindow component and probably not needed here, right?

**Review by philippschwarzmueller — CHANGES_REQUESTED** [review]
pls fix or tell me why i am wrong xdddd

File: frontend/src/components/chatwindow/properties.ts
Comment (LaurinUB) [reply frontend/src/components/chatwindow/properties.ts:1]: maybe 👀

**Review by LaurinUB — COMMENTED** [review]
done

File: backend/package.json
Comment (philippschwarzmueller) [inline backend/package.json:1]: npm install --save @types/cors

**Review by philippschwarzmueller — APPROVED** [review]
:shipit: WAGMI LFG LGTM

**Pull request merged**